### ISSUE TITLE
Questions/suggestions about diffs & closing fugitive:// buffers

### ISSUE THREAD

**Issue by Silex** [tpope/vim-fugitive#81]
Hello,

When I do `:Gstatus`, and go through the modified files looking diffs by pressing `D` (btw the doc doesn't mention it), it opens a lot of diff windows (why additional windows instead of reusing the two?) Then I think ok this is fine, and I close the status window (`q` mapping), but the diffs window stay there. I then go `:bd` or `:q` like crazy, and then I end up doing `:ls` followed by something like `:12,999bd` to close all the fugitive:// buffers that were opened after my buffers (let's say 1-12).

So to sum it up:
1. document the mappings better (D)
2. reuse the diff windows, or provide an option for it
3. Maybe provide something like `:Gbd` that closes all fugitive buffers

Tell me what you think :)
